proj/src/main/webapp/chat.js

```diff
@@ -428,23 +428,29 @@ function initRef(dbRefObject) {
 
 function pushChatMessage() {
     const messageInput = document.getElementById('message-input');
-
-    var message = {
+    // prevent blank messages
+    if (messageInput.value.trim().length != 0){
+        var message = {
         content : messageInput.value,
         timestamp : new Date().getTime(),
         senderDisplay : firebase.auth().currentUser.displayName,
         senderUID : firebase.auth().currentUser.uid
```

Comment:
nit: can you indent this block.

Reply:
for sure; i indented this in the branch i am currently working on